src/Components/Components/test/RendererTest.cs

```diff
@@ -946,7 +1160,7 @@ public void AllRendersTriggeredSynchronouslyDuringEventHandlerAreHandledAsSingle
             Assert.Single(renderer.Batches);
 
             // Act
-            renderer.DispatchEvent(childComponentId, origEventHandlerId, args: null);
+            await renderer.DispatchEventAsync(childComponentId, origEventHandlerId, eventArgs: null);
```

Comment:
I think having an `await` here somewhat undermines the idea of this test, which is about what happens synchronously.

Would it be possible *not* to await this task? Or perhaps capture it and put `await capturedTask` at the end of the test.

Ideally it would be preferable to do that for all the tests being updated here, since we explicitly want the effects to remain synchronous.

Reply:
Yeah good catch. I was trying to preserve the semantics of this but I misread the test name. I'll review these again.